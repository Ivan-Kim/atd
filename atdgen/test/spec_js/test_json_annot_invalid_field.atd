type foo = {
  foo_bar <json nam="fooBar">: bool;
}
