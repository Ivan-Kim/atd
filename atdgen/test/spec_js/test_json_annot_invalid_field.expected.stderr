File "test_json_annot_invalid_field.atd", line 2, characters 16-28:
json.nam annotation not allowed
